webauthn/webauthn.py

```diff
@@ -32,6 +32,11 @@
 
 from . import const
 
+try:
+    from cryptography.hazmat.primitives.asymmetric import ed25519
+except Exception as e:
+    pass
```

Comment:
Doing this as users of this library may not necessarily have or need the version of crytography (i think > 2.6, requires py 3.8) that has ed25519 support.